common/autoconf/libraries.m4

```diff
@@ -670,9 +670,11 @@ AC_DEFUN_ONCE([LIB_SETUP_MISC_LIBS],
   DEFAULT_ZLIB=system
   if test "x$OPENJDK_TARGET_OS" = xwindows; then
     #
-    # On windows default is bundled...on others default is system.
+    # On windows default is bundled.
```

Comment:
Since you're changing this line anyway, please correct the spelling of "Windows".